app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,39 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  #email_notification
+  def reviewer_notification(comment)
     @comment = comment
     @proposal = comment.proposal
 
     # Email all reviewers of this proposal if notifications is true unless they made the comment
-    bcc = @proposal.ratings.map do |rating|
-      user = rating.user
-      if rating.event_teammate.try(:should_be_notified?) && @comment.user_id != user.id
-        user.email
-      end
+    bcc = @proposal.reviewers.with_notifications.map do |reviewer|
```

Comment:
We want to switch to using to addresses, which is fine since this is only to other reviewers.  Also pass in the reviewers as we already have that in the notify method.